### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 13

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#313] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152322640-6b428620-894c-4955-94b7-e831e1e7f11f.png)

**Comment by Exyss** [comment]
L'errore è nel caso base, difatti ponendo N = 1 otteniamo

1 = (1 + 1/2)^2 / 2
1 =  (9/4)/2
1 = 9/8

Che è falso
